frappe/core/doctype/link_field_filter/link_field_filter.json

```diff
@@ -0,0 +1,53 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2023-10-05 17:07:50.711810",
+ "default_view": "List",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "link_field",
+  "field",
+  "condition",
+  "value"
+ ],
+ "fields": [
+  {
+   "fieldname": "link_field",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "label": "Link Field"
+  },
+  {
+   "fieldname": "field",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "label": "Field"
```

Comment:
Maybe "Property"?
"Condition Field" would be possible, too, but in the long run you won't want to restrict this to Fields. In the end, we will want to add non-field document properties as well. So altogether, these are properties. Also we'd avoid the ambiguity of two different "Fields".
Or: "Criterion". That might be super clear, too.